pkg/drivers/kic/oci/network_create.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package oci
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+// firstSubnetAddr subnet to be used on first kic cluster
+// it is one octet more than the one used by KVM to avoid possible conflict
+const firstSubnetAddr = "192.168.49.0"
+
+// big enough for a cluster of 256 nodes
+const defaultSubnetRange = 24
```

Comment:
For clarity, this is a mask rather than a range - which would contain a start and an end. This variable doesn't seem to be overridden, so I feel like the `default` part of the name doesn't carry meaning.

My suggestion would be:

`netMask = net.CIDRMask(24, 32)`

or:

`netMask = 24`

Also, the comment may not be valid: the maximum node usable IP space in a /24 is 254 (first IP is broadcast + 1 for gateway).